[BUG] sendpacket.c:977:5: fatal error: use of undeclared identifier 'mtu'

**Describe the bug**
When trying to compile tcpreplay (https://github.com/appneta/tcpreplay/releases/tag/v4.3.3) from source using android ndk crosscompiling the build fails with:
```
sendpacket.c:977:5: fatal error: use of undeclared identifier 'mtu'
    mtu = ifr.ifr_ifru.ifru_mtu;
    ^
```


**To Reproduce**
Steps to reproduce the behavior:
1. using android-ndk-r21d on Ubuntu 19 x64 to build for aarch64-linux-android
2. `./configure --host=aarch64-linux-android --with-libpcap=/path/to/libpcap/`
3.  make
 
**Expected behavior**
build runs successful

**System (please complete the following information):**
 - OS:  Ubuntu 19 x64 Linux vm 5.3.0-61-generic
 - Tcpreplay Version releases/tag/v4.3.3

**sourced env vars**
export NDK=path/to/toolchains/android-ndk-r21d
export TOOLCHAIN=$NDK/toolchains/llvm/prebuilt/linux-x86_64
export TARGET=aarch64-linux-android
export API=21
export AR=$TOOLCHAIN/bin/$TARGET-ar
export AS=$TOOLCHAIN/bin/$TARGET-as
export CC=$TOOLCHAIN/bin/$TARGET$API-clang
export CXX=$TOOLCHAIN/bin/$TARGET$API-clang++
export LD=$TOOLCHAIN/bin/$TARGET-ld
export RANLIB=$TOOLCHAIN/bin/$TARGET-ranlib
export STRIP=$TOOLCHAIN/bin/$TARGET-strip

Had no time to try it earlier. It now fails with:
```
  CCLD     tcprewrite
./common/libcommon.a(utils.o): In function `_our_safe_realloc':
/home/umrukz3/repos/tcpreplay/src/common/utils.c:76: undefined reference to `rpl_realloc'
./common/libcommon.a(sendpacket.o): In function `sendpacket':
/home/umrukz3/repos/tcpreplay/src/common/sendpacket.c:303: undefined reference to `txring_put'
./common/libcommon.a(sendpacket.o): In function `sendpacket_open_pf':
/home/umrukz3/repos/tcpreplay/src/common/sendpacket.c:980: undefined reference to `txring_init'
../libopts/.libs/libopts.a(libopts_la-libopts.o): In function `ao_realloc':
/home/umrukz3/repos/tcpreplay/libopts/./autoopts.c:78: undefined reference to `rpl_realloc'
clang: fatal error: linker command failed with exit code 1 (use -v to see invocation)

```

had no time to try it again earlier. Used the proposal in https://github.com/appneta/tcpreplay/pull/614
It now fails with:
```
  CC       tcpprep-tcpprep_api.o
tcpprep.c:97:13: fatal error: use of undeclared identifier 'S_IREAD'
            S_IREAD | S_IWRITE | S_IRGRP | S_IWGRP | S_IROTH)) == -1)
            ^
  CC       tcprewrite-tcprewrite_opts.o
1 error generated.
make[3]: *** [Makefile:902: tcpprep-tcpprep.o] Fehler 1

```

--- a/src/defines.h.in
+++ b/src/defines.h.in
@@ -362,7 +362,7 @@ typedef u_int32_t uint32_t
 */
 #ifdef UNUSED 
 #elif defined(__GNUC__) 
-# define UNUSED(x) UNUSED_ ## x __attribute__((unused)) 
+# define UNUSED(x) x __attribute__((unused))
 #elif defined(__LCLINT__) 
 # define UNUSED(x) /*@unused@*/ x 
 #else 

